react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -94,7 +95,7 @@
     };
 
     public ReactAztecText(ThemedReactContext reactContext) {
-        super(reactContext);
+        super(reactContext, AlignmentRendering.VIEW_LEVEL);
```

Comment:
Not a blocker for the PR but, perhaps we'd like to open this up for configuration from the individual blocks that need it. I don't have something particular in mind but, I think it might be possible for a block to need _span level_ alignment.

Reply:
I agree, it wouldn't surprise me at all if we need to set this dynamically in the future.